instance/models/instance.py

```diff
@@ -119,11 +123,13 @@ class Instance(ValidateModelMixin, models.Model):
     # in a query, e.g. to do .select_related('ref_set')
     ref_set = GenericRelation(InstanceReference, content_type_field='instance_type', object_id_field='instance_id')
     openstack_region = models.CharField(
+        # TODO: Move this to OpenedXInstance
         max_length=16,
         blank=False,
         default=default_setting('OPENSTACK_REGION'),
     )
     tags = models.ManyToManyField(
+        # TODO: Move this to InstanceReference (common fields should be in non-abstract tables where possible)
```

Comment:
@bradenmacdonald Yeah, let's see how the refactorings proposed in the app server set design doc pan out, and move the tags when needed.  Shouldn't be a big deal anyway.

I would refrain from uisng tags for any kind of core functionality (whatever "core" means, but you might get the idea).